plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/rest/RefactoringService.java

```diff
@@ -61,6 +61,13 @@ public RefactoringService(RefactoringManager manager) {
         this.manager = manager;
     }
 
+    /**
+     * Create move refactoring session.
+     * @param cmr move settings, contains resource paths to move.
+     * @return refactoring session id.
+     * @throws JavaModelException when JavaModel has a failure
```

Comment:
How client should react on this two Exception. Will he receive plain text responses?